YAXLib/YAXLib.csproj

```diff
@@ -1,64 +1,31 @@
-        <Copyright>Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Copyright>
-        <Authors>Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Authors>
-        <Company></Company>
+        <PackageProjectUrl>https://github.com/MhpSoftware/YAXLib.Redux</PackageProjectUrl>
+        <PackageIcon>YAXLib_256x256.png</PackageIcon>
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <PackageIcon>YAXLib_64x64.png</PackageIcon>
-        <PackageProjectUrl>https://github.com/sinairv/YAXLib</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/sinairv/YAXLib</RepositoryUrl>
-        <RepositoryType>git</RepositoryType>
-        <PackageTags>XML Serialization Serializer Serialize Deserialize .NETFramework .NETStandard .NET</PackageTags>
-        <PackageReleaseNotes>Now supporting netstandard2.0 netstandard2.1 net461 net462 net472 net48 net5.0</PackageReleaseNotes>
-        <OutputType>Library</OutputType>
-        <RootNamespace>YAXLib</RootNamespace>
-        <AssemblyName>YAXLib</AssemblyName>
-        <TargetFrameworks>netstandard2.0;netstandard2.1;net461;net462;net472;net48;net5.0</TargetFrameworks>
-        <LangVersion>8</LangVersion>
-        <WarningLevel>4</WarningLevel>
-        <Nullable>disable</Nullable>
-        <PublishUrl>publish\</PublishUrl>
-        <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-        <DefineConstants>DEBUG;TRACE</DefineConstants>
-        <OutputPath>bin\Debug\</OutputPath>
-        <DocumentationFile></DocumentationFile>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-        <OutputPath>bin\Release\</OutputPath>
-        <DocumentationFile>bin\Release\YAXLib.xml</DocumentationFile>
-        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
-        <Deterministic>true</Deterministic>
-        <PublishRepositoryUrl>true</PublishRepositoryUrl>
-        <EmbedUntrackedSources>true</EmbedUntrackedSources>
-        <IncludeSymbols>true</IncludeSymbols>
-        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
-      <DefineConstants>TRACE</DefineConstants>
+        <PackageTags>XML Serialization Serializer Serialize .NET Core</PackageTags>

```

Comment:
Is this on purpose?

Reply:
`PackageTags` ? These are the NuGet tags.